on_receive_fields.lua

```diff
@@ -116,7 +116,7 @@ function travelnet.on_receive_fields(pos, _, fields, player)
 	end
 
 	if not pos or not fields then
-		player_formspec_data[name] = nil
+		travelnet.actions.end_input(_, _, player)
```

Comment:
Underscore arguments should not be used, if you need argument give name for it and if you do not need then use nil.

Reply:
Good call actually, I should pass these in if they're available, hey might not be used now but there's no reason they couldn't be

Reply:
Passing in as much info as possible now